funzip: add page (#8937)
tldr-pages/tldr: pages/common/funzip.md

# funzip

> > Print the content of the first (non-directory) member in a zip/gz archive without extraction.

Reviewer: ```suggestion
> Print the content of the first (non-directory) member in an archive without extraction.
```

> More information: <https://manned.org/funzip>.

> - Print the content of the first member in an archive:

Reviewer: ```suggestion
- Print the content of the first member in a `.zip` or `.gz` archive:
```

> `funzip {{path/to/archive}}`

Reviewer 2: ```suggestion
`funzip {{path/to/archive.zip}}`
```
Since you are specifying `.zip` archive shouldn't this be `path/to/archive.zip`?

> `funzip {{path/to/archive.zip}}`

Reviewer: add an example description for Gzip files too.

Author: @navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example

Reviewer 2: > @navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example

You can update it right here on GitHub in Files changed tab.

Author: > > @navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example
> 
> You can update it right here on GitHub in Files changed tab.

@kbdharun The edit option is not clickable. Cannot edit or delete the file

Reviewer 2: Same here, I guess you could create a new PR to add this page. In the new PRs description add closes #this-pr-number.

Author: sure, that make sense

Author: @kbdharun , I have created a new PR #9236. We can close this PR

> `funzip {{path/to/archive[.zip|.gz]}}`

Reviewer: ```suggestion
`funzip {{path/to/archive}}`
```

> - Decrypt a `.gz` archive and print the content:

Reviewer: The `-password` doesn't work with Zip?

Author: Should we mentioned `zip` and `gz` in description at all location? the command are same. the only thing that change is extrension.

Author: my bad, it make sense to add `zip` and `gz` is 2nd example only

> `funzip -password {{password}} {{path/to/archive[.zip|.gz]}}`

Reviewer: Well, we have space in this page, split this into 2 examples. Also Gzip only compress 1 file, right? So the `content of the first member` only applies to Zip files.

> `funzip -password {{password}} {{path/to/archive[.zip|.gz]}}`

Author: @navarroaxel,  I made few changes accordingly. Thanks for pointing it out
